python/nav/models/sql/changes/sc.05.04.0002.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO alerttype (eventtypeid,alerttype,alerttypedesc) VALUES
+  ('deviceNotice','deviceFwUpgrade','Firmware upgrade on device.');
```

Comment:
I don't see the other alert types you are referencing from the code here?

Reply:
That is because they already exist in the database

Reply:
Of course they are, silly me 😆